src/main/java/me/karakelley/http/ApplicationFactory.java

```diff
@@ -0,0 +1,19 @@
+package me.karakelley.http;
+
+import me.karakelley.http.filesystem.FileFinderCache;
+import me.karakelley.http.filesystem.PublicDirectory;
+import me.karakelley.http.filesystem.RealFileFinder;
+import me.karakelley.http.handlers.Application;
+import me.karakelley.http.handlers.Handler;
+
+public class ApplicationFactory {
+  private final String NO_DIRECTORY = "nodirectory";
```

Comment:
Hmm. This is a second private constant that has to magically match `CommandLineArguments#DEFAULT_PATH`...